dags/txp_pulse.py

```diff
@@ -0,0 +1,25 @@
+from airflow import DAG
+from datetime import datetime, timedelta
+from operators.emr_spark_operator import EMRSparkOperator
+
+default_args = {
+    'owner': 'harterrt@mozilla.com',
+    'depends_on_past': False,
+    'start_date': datetime(2017, 4, 24),
+    'email': ['telemetry-alerts@mozilla.com', 'harterrt@mozilla.com'],
+    'email_on_failure': True,
+    'email_on_retry': True,
+    'retries': 2,
+    'retry_delay': timedelta(minutes=30),
+}
+
+dag = DAG('txp_pulse', default_args=default_args, schedule_interval='@daily')
+
+t0 = EMRSparkOperator(task_id="txp_pulse",
+                      job_name="Pulse Testpilot Experiment ETL",
+                      execution_timeout=timedelta(hours=2),
+                      release_label="emr-5.0.0",
```

Comment:
Please use the default EMR release unless there is a strong reason to hardcode it.